msteams-platform/bots/how-to/format-ai-bot-messages.md

```diff
@@ -0,0 +1,459 @@
+
+| Feature | Bot | AI bot |
+| --- | :---: | :---: |
+| [AI label](#ai-label) that enables user to identify that the message was generated using AI. | ❌ | ✔️ |
+| [Sensitivity label](#sensitivity-label) that enables user to understand the confidentiality of the bot's message. | ❌ | ✔️ |
+| [Citations](#citations) that enables user to refer to the source of the bot's message through in-text citations and a list of references. | ✔️ | ✔️ |
+| [Feedback buttons](#feedback-buttons) that enables user to provide positive or negative feedback based on their experience. | ✔️ | ✔️ |
+
+:::image type="content" source="../../assets/images/bots/ai-bot-message.png" border="false" alt-text="Screenshot shows the AI-generated bot message with the respective AI bot message features.":::
+
+> [!NOTE]
+>
+> AI label, sensitivity label, citations, and feedback buttons:
+>
+> * Are available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> * Aren't supported in message extensions.
+
+## AI label
+
+AI label added to your bot message specifies that the message is generated by AI. AI bots use large language models (LLMs) that are generally reliable, however, it's crucial to add an AI label to maintain user's trust on your bot.
+
+To add an AI label to your bot message, modify the message to include `additionalType` field under the `entities` object.
+
+The following code snippet shows how to add an AI label to your bot's message:
+
+```json
+{ 
+    "type": "message", 
+    "from": { 
+        "id": "28:48b2e1fd-b6b8-46ce-8074-8e918304fcb2" 
+    }, 
+    "conversation": { 
+        "isGroup": true, 
+        "id": "19:88b15596-2dd4-440b-98c6-a19cf49455a1@thread.tacv2" 
+    }, 
+    "recipient": { 
+        "id": "8:orgid:f20c2a2c-1b43-416a-b5e8-fbcc69387c5e" 
+    }, 
+    "text": "2fd50566-ad63-4237-b240-f1942f7bdeb6", 
+    "attachments": [], 
+    "entities": [ 
+        { 
+            "@type": "Message", 
+            "@context": "https://schema.org", 
+            "@id": "", 
+            "additionalType": [ 
+                "AIGeneratedContent" 
+            ] 
+        } 
+    ] 
+}
+```
+
+For a bot built using Teams AI library, Teams enables AI label to all AI-powered bot messages when `additionalType` field is added under `entities` object.
+
+```javascript
+await context.sendActivity({
+         type: ActivityTypes.Message,
+         text: `Hey I'm a friendly AI bot. This mesasge is generated via AI - ${txt}`,
+        entities: [
+          {
+            type: "https://schema.org/Message",
+            "@type": "Message",
+            "@context": "https://schema.org",
+            additionalType: ["AIGeneratedContent"], // AI Generated label
+          }
+        ]
+       });
+```
+
+After an AI label is added, your bot’s message automatically display an **AI-generated** label next to the bot’s name with a hover disclaimer stating **AI-generated content may be incorrect**.
```

Comment:
Immediate next questions are, is this localized? Can I customize it?

Reply:
We have updated the content to reflect that the label and disclaimer are not mofidiable.